apps/AppWithWearable/lib/flavors.dart

```diff
@@ -0,0 +1,21 @@
+enum Flavor {
+  prod,
+  dev,
+}
+
+class F {
```

Comment:
IMO, I will consider writing `Environment` rather than only `F` as a class name. Naming convention can be improved here. 

Personally, I will write something like this:
```dart
enum Environment {
  staging,
  production,
  stub,
  testing,
}

extension EnvironmentExt on Environment {
  String get label {
    switch (this) {
      case Environment.staging:
        return 'staging';
      case Environment.stub:
        return 'stub';
      case Environment.testing:
        return 'test';
      default:
        return '';
    }
  }
}
```